tests/unit/droppable/droppable_events.js

```diff
@@ -60,4 +58,43 @@ test("drop", function() {
 });
 */
 
+test( "drop event callback order", function() {
+	expect( 4 );
+
+	$( "<div id='droppable3'></div>" ).appendTo( "body" );
+	$( "<div id='droppable4'></div>" ).appendTo( $( "#droppable3" ) );
+	$( "<div id='draggable2'></div>" ).appendTo( "body" );
+	var droppable3 = $( "#droppable3" ),
+		droppable4 = $( "#droppable4" ),
+		draggable2 = $( "#draggable2" );
```

Comment:
Why not just store the references as we create the elements? They don't even need IDs then.